<launch>
  <node pkg="point_cloud_mapping" type="pcd_to_msg_node" name="pcd_to_msg_node" output="screen" respawn="true"
	args="$(optenv pcd /home/marton/work/datasets/NICO/teapot_at0.pcd) $(optenv period 5)" />

  <node pkg="cloud_algos" type="rotational_estimation_node" name="rotational_estimation_node" output="screen" respawn="true">
    <remap from="~cloud_pcd" to="cloud_pcd" />
  </node>


<!--
 <node pkg="cloud_tools" name="msg_to_pcd" type="msg_to_pcd" respawn="true" output="screen">
   <param name="input_cloud_topic" value="/mls_fit_node/cloud_mls"/>
   <param name="dir" value="$(env HOME)/work/datasets/TMP/"/>
   <param name="nr_saved_pcds" type="int" value="0"/>
 </node>
-->

</launch>
